package.json

```diff
@@ -13,7 +13,6 @@
   ],
   "scripts": {
     "start": "node ./bin/http-server",
-    "pretest": "common bin/http-server lib/ test",
```

Comment:
Style checking as part of testing is very important; that way the code is consistently readable. So while `common-style` is deprecated, I think it should be replaced rather than removed.

Reply:
I've made PR #492 for discussion with regards to `common-style` and friends.